libcontainer/configs/namespaces_syscall.go

```diff
@@ -19,6 +19,12 @@ var namespaceInfo = map[NamespaceType]int{
 	NEWCGROUP: unix.CLONE_NEWCGROUP,
 }
 
+// These flags must be the same according to
+// libcontainer/nsenter/namespace.h
+var namespaceNonCloneInfo = map[NamespaceType]int{
+	NEWIMA: 0x80000000,
```

Comment:
1. cross-i386 CI job complains:

> cannot use 0x80000000 (untyped int constant 2147483648) as int value in map literal (overflows)

So I guess we have to use explicit uint64 here.

2. I'd rather not use a map here, but write a function with a switch statement, somewhat similar to HasProcfs above, mostly because a map initialization incurs some init overhead, and in case of a single element it is not worth it.

Reply:
Or maybe `uint32`, or `uint` (depending on what type the kernel is actually using for this).

Reply:
... and modify bootstrapData (both sides) and data structures accordingly

Reply:
> depending on what type the kernel is actually using for this

The kernel uses nothing for this (as I've written there's no `CLONE_*` or any other similar flags for IMA). This value (`0x80000000`) is just the next bit after `CLONE_NEWNET`. _runC_ can decide to use any flag or even not use a flag.

> I'd rather not use a map here, but write a function with a switch statement

Interesting idea, I'll try to modify code this way.

 Thanks!